onnxscript/function_libs/torch_lib/graph_building/_graph_building_torch.py

```diff
@@ -730,14 +731,7 @@ def _add_constant_to_graph(self, constant) -> torch.Value:
         value.setDebugName(_rename_intermediate_value(value.debugName()))
         return value
 
-    @runtime_typing.checked
-    def _add_torchscript_op_call(
-        self,
-        name: str,
-        onnx_inputs: Sequence[ValidInputType],
-        onnx_attributes: Mapping[str, ValidArgumentType],
-        n_outputs: int,
-    ) -> Union[TorchScriptTensor, Tuple[TorchScriptTensor, ...]]:
+    def preprocess_inputs(self, onnx_inputs: Sequence[ValidInputType]) -> List[torch.Value]:
```

Comment:
```suggestion
    def _preprocess_inputs(self, onnx_inputs: Sequence[ValidInputType]) -> List[torch.Value]:
```

Reply:
This will trigger `protect-access` to line:

```python
inputs = self._graph.preprocess_inputs(inputs)
```

Reply:
I see. then keeping the original is ok